engine/setup/src/include/file.cpp

```diff
@@ -167,7 +167,7 @@ void LoadConfig( void )
         n_parm = next_parm( parm );                     //next_parm is a line splitter included with general.c
         if (strcmp( "groups", parm ) == 0) {
             parm = n_parm;
-            while ( ( n_parm = next_parm( parm ) ) > 0 ) {
```

Comment:
Please verify this is actually doing pointer comparison against zero and not getting evaluated as something else first.

Reply:
See response below

Reply:
I didn't actually see a response from Nachum, but it sure looks like it's pointer comparison against zero.
n_parm is of type char *. next_parm gets the next parameter from the string, sorted by a space, sticks a \0 at the space and returns a pointer to the right side. I'm actually refactoring in this area and may well replace everything here with a line or two from boost.